base_wamas/data/ir_config_parameter_data.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
```

Comment:
@tuantrantg for data, we should make them `noupdate`

Reply:
hi a @dzungtran89 I'll set `noupdate="1"` to all the files in folder `data` in the upcoming commit